docs/TrafficSplitting.md

```diff
@@ -0,0 +1,157 @@
+# Traffic Splitting
+
+The purpose of this section of the kn documentation is to describe the command verbs for traffic splitting operations and present some examples.
+
+`kn` client can work with following properties of a traffic target defined in traffic block of a service spec.
+ - **RevisionName**: RevisionName of a specific revision to which to send a portion of traffic.
```

Comment:
Nit.. Do we need to repeat the word when it's already bolded and delimits the paragraph :) ?

Reply:
Would also rephrase to '**RevisionName**: Name of a specific revision ...'

Reply:
sure, we'll have this incorporated in a different PR.